packages/hoppscotch-common/src/components/http/ResponseMeta.vue

```diff
@@ -1,7 +1,16 @@
 <template>
   <div
-    class="sticky top-0 z-10 flex items-start justify-center flex-shrink-0 p-4 overflow-auto overflow-x-auto bg-primary whitespace-nowrap"
+    class="sticky top-0 z-10 flex items-center justify-center flex-shrink-0 p-4 overflow-auto overflow-x-auto bg-primary whitespace-nowrap"
   >
+    <AppInspection
+      v-if="response?.type !== 'loading'"
+      :inspection-results="tabResults"
+      :class="[
+        response === null || response?.type === 'network_fail'
+          ? 'absolute right-0 top-0'
+          : 'mr-2',
```

Comment:
Can you share a screenshot of these cases?
1. `'absolute right-0 top-0'`
2. `'mr-2'`

I think it'd be better if we put the alert icon on the right-hand side. So you might want to change 2. to `'ml-2'` as a safe margin between the Inspections component and Response's meta.